tests/beaker_tests/cisco_ip_multicast/test_ip_multicast.rb

```diff
@@ -37,6 +37,9 @@
 # Skip -ALL- tests if feature is not supported by image version
 skip_nexus_image(/I[2-6]/, tests)
 
+# Default values changed between I7.3 and I7.4
```

Comment:
Nice!  I was literally setting up my environment to work on this issue and saw your PR.

This change in default behavior is also present in our latest Hamilton release which is `NXOS: version 9.2(1)` so this should handle that case as well.

Reply:
Thinking more, this will not handle our older releases.  It should also be false for

`7.0(3)I2, 7.0(3)I3, 7.0(3)I4, 7.0(3)I5, 7.0(3)I6`

Reply:
The tests are skipped entirely for I2-6.  Support for the feature was added in I7.

Reply:
Ah, good point. Thanks for the reminder.